core/src/androidTest/java/org/hisp/dhis/android/core/common/CategoryComboMockFactory.java

```diff
@@ -0,0 +1,26 @@
+package org.hisp.dhis.android.core.common;
+
+
+import android.support.annotation.NonNull;
+
+import org.hisp.dhis.android.core.category.Category;
+import org.hisp.dhis.android.core.category.CategoryCombo;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class CategoryComboMockFactory {
```

Comment:
this class is not a factory, it's an [object mother](https://martinfowler.com/bliki/ObjectMother.html), can you refactor it?

Reply:
Sure!